docs/guides/use_checkpointing.md

```diff
@@ -193,46 +221,52 @@ serialization.from_state_dict(custom_target, raw_state_dict)
 
 Checkpointing is I/O heavy, and if you have a large amount of data to save, it may be worthwhile to put it into a background thread, while continuing with your training.
 
-You can do this by creating an `async_manager` (as demonstrated in the code cell below) and let it track your save thread.
+You can do this by creating an `orbax.AsyncCheckpointer` (as demonstrated in the code cell below) and let it track your save thread.
 
-`async_manager` is a parameter in [`flax.training.checkpoints.save_checkpoint`](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#flax.training.checkpoints.save_checkpoint) with the default setting at `None`.
+Note that you should use the same `async_checkpointer` to handle all your async saves across your training steps, so that it can make sure that a previous async save is done before the next one begins. This allows bookkeeping like `keep` and `overwrite` to be consistent across steps.
```

Comment:
Suggestion:

```suggestion
Note: You should use the same `async_checkpointer` to handle all your async saves across your training steps, so that it can make sure that a previous async save is done before the next one begins. This enables bookkeeping, such as `keep` and `overwrite` to be consistent across steps.
```